.devcontainer/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/python:3
```

I'd suggest pinning to a base OS/distro so that this this dev container continues to work even if the upstream OS changes (this happened recently with [the new version of debian](https://github.com/devcontainers/images/issues/607).  We often update our images fairly quickly with new distro releases, some upstream tools take a bit of time.

```suggestion
FROM mcr.microsoft.com/devcontainers/python:3-bullseye
```

or
```suggestion
FROM mcr.microsoft.com/devcontainers/python:3-bookworm
```

---

Ouuu thank you! Will do!